src/lib/pseudo.js

```diff
@@ -1,27 +1,45 @@
 'use strict';
 
-// Recursively walk an AST node searching for content leaves
-export function walkContent(node, fn) {
-  if (typeof node === 'string') {
-    return fn(node);
+// Walk an entry node searching for content leaves
+export function walkEntry(entry, fn) {
```

Comment:
We still want to recurse into `value` and `attrs`.  Maybe just filter out nodes with `type` defined which would exclude expressions and indexes?